### DISCUSSION TITLE
Wiiu pro controller changing to Wiimote

### DISCUSSION THREAD

**Discussion by coolwormgit** [KeyPuncher/WiinUPro#161] [Help]
Hello, i have 2 Wiiu pro controller and everything is working great, but when i run Retrobat is changing to wiimote and controllers stop working. Is there any solution to lock controller to Wiiu pro??

Thanks

**Comment by KeyPuncher** [comment]
That's really weird. Does Retrobat have native support for Wiimotes?

**Reply by coolwormgit** [reply]
no, its using SDL. I have to select older sdl or i found a temp solution one app Hidhide and i hide the Nintendo controller.
